server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/EmailSubmissionSet.scala

```diff
@@ -122,7 +122,9 @@ case class EmailSubmissionId(value: Id)
 
 case class EmailSubmissionCreationResponse(id: EmailSubmissionId)
 
-case class EmailSubmissionAddress(email: MailAddress)
+case class Parameters(parameters: Map[String, String])
+
+case class EmailSubmissionAddress(email: MailAddress, parameters: Parameters)
```

Comment:
We can IMO make parameters optional, this should allows us not to change the `EmailSubmissionSetMethod` code:

```suggestion
case object Parameters {
   val EMPTY = Parameters(Map())
}

case class Parameters(parameters: Map[String, String])

case class EmailSubmissionAddress(email: MailAddress, parameters: Parameters = EMPTY)
```

Then we can keep `EmailSubmissionAddress(mailAddress)` and limit the boiler plate...